File: test_archives/py_functions/batch3/function2.py
def function2(string1, int1, float1):
  '''
  This function takes three data types as parameters: a string, an integer, and a float.
  It returns a single integer value.
  '''
  string1 = str(string1)
  int1 = int(int1)
  float1 = float(float1)
  
  result = len(string1) * int1 * float1
  return int(result)

File: test_archives/cpp_functions/batch3/function2.cpp
#include <string>
#include <cmath>

using namespace std;

int function2(string string1, int int1, float float1){
  string1 = to_string(string1);
  int1 = static_cast<int>(int1);
  float1 = static_cast<float>(float1);

  float result = string1.length() * int1 * float1;
  return static_cast<int>(result);
}